src/database/migrations/20231220235718_RemoveParticipantIdColumnFromAuditTrail.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex('auditTrails')
+    .whereNotNull('participantId')
+    .update({
+      eventData: knex.raw("JSON_MODIFY(eventData, '$.participantId', ?)", [
+        knex.ref('participantId'),
+      ]),
+    });
+
+  await knex.schema.alterTable('auditTrails', (table) => {
+    table.dropForeign('participantId');
+    table.dropColumn('participantId');
+  });
+}
+
+export async function down(knex: Knex): Promise<void> {
```

Comment:
Technically this `down` method is missing the fact that `participantId` should be removed from `eventData`. Should we include that?

Reply:
Yeah good point. Have added a call for this.